apps/shopify/src/utils/base64.js

```diff
@@ -0,0 +1,25 @@
+export const convertBase64ToString = (str) => {
+  try {
+    const decodedId = window.atob(str);
+    return decodedId;
+  } catch (error) {
+    return null;
+  }
+};
+
+export const convertStringToBase64 = (str) => {
+  try {
+    const undecodedId = window.btoa(str);
+    return undecodedId;
+  } catch (error) {
+    return null;
+  }
+};
+
+export const convertIdToBase64 = (res) => {
```

Comment:
Can we improve that name. We do not really only convert the id. We convert the nested product as well